cuda_core/tests/test_memory.py

```diff
@@ -126,6 +127,24 @@ def is_host_accessible(self) -> bool:
         return False
 
 
+def test_package_contents():
+    expected = [
+        "Buffer",
+        "MemoryResource",
+        "DeviceMemoryResource",
+        "DeviceMemoryResourceOptions",
+        "IPCBufferDescriptor",
+        "IPCAllocationHandle",
+        "LegacyPinnedMemoryResource",
+        "VirtualMemoryResourceOptions",
+        "VirtualMemoryResource",
+    ]
+    d = {}
+    exec("from cuda.core.experimental._memory import *", d)  # noqa: S102
+    d = {k: v for k, v in d.items() if not k.startswith("__")}
```

Comment:
Q: We should exclude everything starting with one underscore, not just those with two?